<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wyaWidget</class>
 <widget class="QWidget" name="wyaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wya</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="authWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>216</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLineEdit" name="loginLineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Логин</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="passwordLineEdit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>Пароль</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="loginPushButton">
               <property name="text">
                <string>Войти</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="registerPushButton">
               <property name="text">
                <string>Зарегистрироваться</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>215</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mainWidget">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="searchLineEdit">
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>450</width>
               <height>35</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Поиск</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>450</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QTabBar::tab {min-width: 225px;max-width: 400px;}</string>
             </property>
             <property name="tabPosition">
              <enum>QTabWidget::North</enum>
             </property>
             <property name="tabShape">
              <enum>QTabWidget::Rounded</enum>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <property name="elideMode">
              <enum>Qt::ElideNone</enum>
             </property>
             <property name="usesScrollButtons">
              <bool>true</bool>
             </property>
             <property name="documentMode">
              <bool>true</bool>
             </property>
             <property name="tabsClosable">
              <bool>false</bool>
             </property>
             <property name="movable">
              <bool>false</bool>
             </property>
             <property name="tabBarAutoHide">
              <bool>false</bool>
             </property>
             <widget class="QWidget" name="friendsTab">
              <attribute name="title">
               <string>Друзья</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QPushButton" name="addFriendPushButton">
                 <property name="text">
                  <string>Добавить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="groupsTab">
              <attribute name="title">
               <string>Группы</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QPushButton" name="addGroupPushButton">
                 <property name="text">
                  <string>Добавить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>30</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
